Doc/library/concurrent.futures.rst

```diff
@@ -124,10 +124,11 @@ And::
    executor.submit(wait_on_future)
 
 
-.. class:: ThreadPoolExecutor(max_workers=None, thread_name_prefix='')
+.. class:: ThreadPoolExecutor(max_workers=None, thread_name_prefix='', max_queue_size=0)
 
    An :class:`Executor` subclass that uses a pool of at most *max_workers*
-   threads to execute calls asynchronously.
+   threads to execute calls asynchronously. If all worker threads are busy it
+   buffers new work items using a queue of maximum size *max_queue_size*.
```

Comment:
Please document that `map` and `submit` will block if the queue size is reached.